network-device-names.cfg

```diff
@@ -0,0 +1,9 @@
+PREFIX="ens"
+SUFFIX=""
```

Comment:
We are sourcing this to a main configuration file.
Please add `NET_` prefix to these variables to avoid collisions.

Reply:
Good point, fixed.